site/docs/components/component_overview.mdx

```diff
@@ -33,21 +35,33 @@ The React components and technical documentation are available in [Helsinki Desi
 
 ## Component status
 
-Since HDS is still in alpha stage, the component libraries are still experimental. The existing components can be used in projects but changes may still occur.
+**Since HDS is still in alpha stage, the component libraries are still experimental. The existing components can be used in projects but changes may still occur even for stable components.**
+
+HDS uses a simple labeling system for components to communicate their completeness and susceptibility to changes:
+
+Status                                               | Description
+-----------------------------------------------------|--------------
+<StatusLabel type="alert">Pre-release</StatusLabel>  | Component may be missing some features and/or is subject to breaking changes. These components are also open for feedback. **Avoid using in production.**
+<StatusLabel type="info">Stable</StatusLabel>        | Component's implementation, design and documentation are stable and fully featured. **These components can be used in production.** **In the alpha stage we reserve a possibility to do breaking changes to stable components but this is unlikely!** 
+<StatusLabel type="success">Accessible</StatusLabel> | Component has been tested for accessibility by a 3rd party auditor.
+<StatusLabel>New</StatusLabel>                       | Component has been released in or after the latest major release.
+<StatusLabel type="error">Updated</StatusLabel>      | A breaking change has been introduced to this component in the latest major release.
+<StatusLabel type="error">Deprecated</StatusLabel>   | Component will be removed in the upcoming major releases. **Avoid using in production.**
 
 For future components, see [Roadmap](/about/roadmap).
 
 Component                                                        | Status       | Core                                                    | React
 -----------------------------------------------------------------|--------------|---------------------------------------------------------|----------------------------------------------------------------------
```

Comment:
Should HDS IconCheck be used instead of custom checkmarks?

Reply:
Fixed.